edgelet/version.txt

```diff
@@ -1 +1 @@
-1.2.5
+1.2.7
```

Comment:
Nice! We always have the edgelet version the same as the edgeAgent & edgeHub version (even if that means skipping a version number).